packages/serverpod/lib/src/database/database.dart

```diff
@@ -206,7 +207,9 @@ class Database {
 
   /// Executes a single SQL query. A [List] of rows represented of another
   /// [List] with columns will be returned.
+  ///
   /// You are responsible to sanitize the query to avoid SQL injection.
+  @Deprecated('use [query] instead.')
```

Comment:
Suggestion: Keep `unsafeQuery(...)` and `unsafeExecute(...)`.

I understand your motivation for thinking about removing the prefix in the method name. Since we now provide query parameters, it is possible to perform a query in a sanitized way. This, however, does not remove the responsibility from the developer to actually use it correctly.

It is still possible to input any String in the query, and we won't do any validation to warn the developer if they do something incorrect. As opposed to our other database methods where we enforce stricter rules.

Therefore, I would keep the name, highlighting the extended responsibility around using it and the means (QueryParameters) we provide to help out with input sanitizing.

Reply:
Remember to update the examples in `QueryParameters` as well.

Reply:
Hmmmm. I'd really like to deprecate the unsafe ones, because it will make people look at them again and maybe refactor to use parameters. 

Using parameters in queries is pretty much standard and now that we provide them denoting them as unsafe seems to be overly cautious. But if you insist, I'll change it back to use unsafe naming.

Reply:
Thanks, I understand your thought. The cautiousness comes from knowing how much experience many of our users have from backend development.

It is a small cost to pay to highlight this a bit extra.

We will also make sure to highlight this in the update notes (although not as efficient as deprecating the old method).

Reply:
...sorry to ask dumb questions to avoid misunderstanding, this means to keep the deprecation?

Reply:
I assumed now  it means to keep them. This yields quite a few warnings for existing code that deprecated functions are being used.

Reply:
NP!

No, please remove the deprecation and revert the names back to `unsafeQuery(...)` and `unsafeExecute(...)`.